pkg/scheduler/server/internal/cron/cron.go

```diff
@@ -222,9 +223,19 @@ func (c *cron) HostsWatch(stream schedulerv1pb.Scheduler_WatchHostsServer) error
 	}
 }
 
-func (c *cron) triggerJob(ctx context.Context, req *api.TriggerRequest) *api.TriggerResponse {
+func (c *cron) triggerHandler(ctx context.Context, req *api.TriggerRequest) *api.TriggerResponse {
 	log.Debugf("Triggering job: %s", req.GetName())
```

Comment:
Should this log line be deleted since you added the one below? Otherwise we log twice for triggering the job - unless that was intentional?
```suggestion
```

Reply:
Yes it's intentional, so we get a _debug_ log, before and after we have completed execution.